config/example.yml

```
@@ -39,7 +39,7 @@ nelson:
     # These options are passed to a Symfony Finder to find your original translation files.
     # Be careful, only select your origin language files (for example, only *.en.yml files).
     finder_options:
-        in: 'src/'
+        in: 'src/' # This can be an array of folders
```

```suggestion
        in: ['src/']
```
Making it array by default would simplify the code and remove the need for comment